ext/exception.cpp

```diff
@@ -58,7 +58,7 @@ namespace Tango
 
 void sequencePyDevError_2_DevErrorList(PyObject *value, Tango::DevErrorList &del)
 {
-    long len = max((int)PySequence_Size(value), 0);
+    long len = std::max((int)PySequence_Size(value), 0);
```

Comment:
I see the AppVeyor build is failing [here](https://ci.appveyor.com/project/tiagocoutinho/pytango/builds/36759949/job/ly1bhcvmwlln2dhx?fullLog=true#L994) - the Visual Studio compiler doesn't like this.  Any idea why?  Can we leave this one out, or do we need an `#ifdef`?

Reply:
It's an old answer but from https://stackoverflow.com/a/2789509, it looks like we need to use `#define NOMINMAX` for windows.
I'll give it a try.

Reply:
I pushed a new commit that should fix the build on windows.